src/global.css

```diff
@@ -0,0 +1,49 @@
+body {
+  color: #fff;
+  background: #17181c;
+  font-family: sans-serif;
+}
+.wrapper {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+header {
+  padding: 10px 30px;
+}
+
+header h1 {
+  font-size: 40px;
+  font-weight: 800;
+}
+main {
+  flex: 1 1 auto;
+}
+
+footer {
+  height: 100px;
+}
+footer .copyright {
+  font-size: 20px;
+  color: #333;
+  text-align: center;
+}
+footer .copyright a {
+  color: #444;
+}
+footer .copyright a:hover {
+  color: #555;
+}
```

Comment:
It is better to preserve consistency and add empty lines between CSS rules.

Reply:
Fixed